portal/views/portal.py

```diff
@@ -66,7 +66,29 @@ def landing():
         return next_after_login()
 
     timed_out = request.args.get('timed_out', False)
-    return render_template('landing.html', user=None, no_nav="true", timed_out = timed_out)
+    gil = current_app.config.get('GIL', None)
```

Comment:
The return from a dict.get('key') call where no key is found is None by default.  i.e. this can be simply:
`gil = current_app.config.get('GIL')`